w3af/core/ui/api/main.py

```diff
@@ -24,17 +24,13 @@
 
 from w3af.core.ui.api import app
 from w3af.core.ui.api.utils.cli import process_cmd_args_config
-from w3af.core.controllers.dependency_check.dependency_check import dependency_check
-
+from w3af.core.ui.api.utils.digital_certificate import SSLCertifiacate
 
 def main():
     """
     Entry point for the REST API
     :return: Zero if everything went well
     """
-    # Check if I have all needed dependencies
-    dependency_check()
-
```

Comment:
Why this change? I believe you did something wrong with the merge/pull request. You should base your REST API changes on my develop, and then send a PR to the develop branch.